src/modules/fstab/main.py

```diff
@@ -152,20 +152,22 @@ def generate_crypttab_line_info(self, partition):
         if not mapper_name or not luks_uuid:
             return None
 
-        if mount_point == "/":
+        if mount_point != "/":
             return None
 
         return dict(
             name=mapper_name,
             device="UUID=" + luks_uuid,
             password="/crypto_keyfile.bin",
+            options="luks,keyscript=/bin/cat",
```

Comment:
options="luks" should probably be set on all distros. The "keyscript=/bin/cat" thing seems to be Debian-specific (including of course Debian derivatives) though, see:
http://www.pavelkogan.com/2015/01/25/linux-mint-encryption/
which tracks this down to the `cryptroot` script from the Debian `initramfs-tools`.

So I think this will need to be optional. Maybe add a `crypttabOptions: luks` line to fstab.conf, which you can customize to `crypttabOptions: luks,keyscript=/bin/cat` on Debian? (Of course, you also need to add the Python code to actually read the option.)